### ISSUE TITLE
[??] BE-002 ????? pipeline

### ISSUE THREAD

**Issue by ioir123ju** [ioir123ju/visibility-detection#2]
## ????
?????????:resize?normalize?????(??/??/???)?

## ????
- [ ] preprocess.py ?????
- [ ] ?? resize ? 224x224,???? ImageNet ????
- [ ] ?????????????
- [ ] ?? PyTorch DataLoader ??

## ????
REQ-002
